template/message.ts

```diff
@@ -15,7 +15,7 @@ interface bodyValid {
   }
 }
 
-const messageValid = (payload: bodyValid) => {
+const messageValid = (payload: bodyValidInterface) => {
```

Comment:
Mising return type data in function

Reply:
thanks i forgot to return type data :smile: :+1: